.golangci.yaml

```diff
@@ -1,5 +1,5 @@
 run:
-  deadline: 10m
+  go: "1.23"
```

Comment:
Hmm, did we not remove these go version configs elsewhere?

Reply:
in other repos, it was here already, this was missing in bmo it seems.